### PR TITLE
plugins:puppetdb_foreman - Release 0.2.0

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

File: rubygem-puppetdb_foreman/rubygem-puppetdb_foreman.spec [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 Summary:    Foreman plugin to interact with PuppetDB
 Name:       %{?scl_prefix}rubygem-%{gem_name}
-Version:    0.1.3
+Version:   0.2.0
```
Comment (domcleal) [inline rubygem-puppetdb_foreman/rubygem-puppetdb_foreman.spec]: Nitpick, the indentation's not right here.

**Comment by domcleal** [comment]
Which repos did you want this in?

**Comment by dLobatog** [comment]
@domcleal It should work with anything 1.4+, but I don't think you're building for anything earlier than 1.8, right? 1.8 1.9 1.10 then

**Comment by domcleal** [comment]
Yep, thanks @dLobatog.  Building for those three plus nightly.

**Pull request merged**